composer.json

```diff
@@ -29,6 +29,7 @@
     },
     "require-dev": {
         "facade/ignition": "^2.5",
+        "guzzlehttp/guzzle": "^7.3",
```

Comment:
Do we need to add this explicitly? Won't this dependency already be required testbench?

Reply:
Sadly not. Laravel doesn't include Guzzle out of the box, you have to install it separately if you want to use the Http Client: https://laravel.com/docs/master/http-client#introduction and Orchestra is no different.